pkg/kubectl/cmd/cp.go

```diff
@@ -164,6 +164,9 @@ func (o *CopyOptions) Validate(cmd *cobra.Command, args []string) error {
 }
 
 func (o *CopyOptions) Run(args []string) error {
+	if len(args) < 2 {
+		return fmt.Errorf("Unexpected args: %v", args)
```

Comment:
I think suggesting to the user what is required might make more sense:
`a source and a destination are required` or something along those lines